Proiect1/Proiect1.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
you have binaries on git!!!

Reply:
Initially I didn't understand what you meant so I deleted the .exe and .dll files but then I remembered that it was about .gitignore. I added it and everything should be ok now